projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -236,10 +240,13 @@ export class MultiSelectComponent<V> implements AfterContentInit, OnChanges {
     );
     if (selectedItems === undefined || selectedItems.length === 0) {
       this.triggerLabel = this.placeholder;
+      this.selectedItemsCount = 0;
```

Comment:
nit: can move this assignment out of the condition then collapse the conditional

```ts
this.selectedItemsCount = selectedItems?.length ?? 0;
this.triggerLabel = this.selectedItemsCount === 0 ? this.placeholder : selectedItems[0].label;

Reply:
updated